bindings/python/tests/feature/test_feature.py

```diff
@@ -136,6 +136,36 @@ def test_feature_location_setters(self):
         self.assertEqual(test_feature_0.location.end.column_offset,
                          18)
 
+        test_feature_0.location = test_feature_0.location
+        self.assertTrue(test_feature_0.location)
+        self.assertEqual(test_feature_0.location.path, path)
+        self.assertEqual(test_feature_0.location.start.line_number,
+                         4)
+        self.assertEqual(
+            test_feature_0.location.start.column_offset, 2)
+        self.assertEqual(test_feature_0.location.end.line_number, 4)
+        self.assertEqual(test_feature_0.location.end.column_offset,
+                         18)
```

Comment:
This test code looks like the test case bellow. Does this make sense here?
If it does, this should be a different test case which specifically checks for self assignment for location setters.